copy-db.js

```diff
@@ -105,7 +105,7 @@ console.log(`Building user roles...`);
 
 console.log(`Importing database snapshot...`);
 run(`docker cp ${DUMP_FILE} ${IMAGE_NAMES.POSTGRES}:/`);
-postgres(`pg_restore ${DB_FLAGS} -dwagtail ${DUMP_FILE}`);
+postgres(`pg_restore ${DB_FLAGS} -dwagtail --no-owner ${DUMP_FILE}`);
```

Comment:
Thanks for leading this @danielfmiranda. 

I would suggest linking to the Heroku docs for future debugging reference and also adopt the `--no-acl` flag (I guess they know why it's in their docs). 

```suggestion
// Based on the Heroku docs for restoring to local database:
// https://devcenter.heroku.com/articles/heroku-postgres-import-export#restore-to-local-database
postgres(`pg_restore ${DB_FLAGS} -dwagtail --no-acl --no-owner ${DUMP_FILE}`);
```